inst/odin/carehomes.R

```diff
@@ -113,7 +120,12 @@ prob_admit_conf[] <- p_admit_conf * psi_admit_conf[i]
 
 ## Draws from binomial distributions for numbers changing between
 ## compartments:
-n_SE[, ] <- rbinom(S[i, j], p_SE[i, j])
+
+# competing risk of infection and progression through vaccine classes
+n_S_out[, 1:n_vacc_classes_minus_1] <- rbinom(S[i, j], p_SE[i, j] + p_SS[i, j])
```

Comment:
Surely the SE and SS transitions are not mutually exclusive? Unless I'm mistaken, this calculation doesn't seem to account for the intersection of these transitions.

Reply:
agreed - I think for a small enough timestep it is ok to assume no-one gets vaccinated and infected in the same time step (or changes vaccination category and gets infected in the same timestep). @edknock and @MJomaba thoughts about this?

Reply:
Happy now with how this has been revised.